Source/SpacePort/DBAssets/DbModels/PersonModel.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using System.Threading.Tasks;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace SpacePort
+{
+    public class PersonModel
+    {
+        [Key]
+        public int PersonID { get; private set; }
+        [MaxLength(50)]
```

Comment:
Why a max length of 50?

Reply:
Dels för att testa, dels för att inte av misstag råka lägga in gigantiska strängar. Just 50 är bara en siffra egentligen.